environment/.deployment/production/docker-compose.production.yml

```
@@ -0,0 +1,79 @@
+networks:
+  traefik-proxy:
+    external: true
+  lmt-production:
+    name: lmt-production
+
+volumes:
+  lmt-production-postgres-data:
+    name: lmt-production-postgres-data
+  lmt-production-storage-data:
+    name: lmt-production-storage-data
+
+services:
+  lmt-production-app:
+    image: registry.blumilk.pl/internal-public/lmt:latest
+    container_name: lmt-production-app-container
+    pull_policy: always
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "50m"
+        max-file: "5"
+    deploy:
+      mode: replicated
+      replicas: 1
+      resources:
+        limits:
+          memory: 1GB
+    labels:
+      - "io.portainer.accesscontrol.teams=LMT-production"
+      - "traefik.enable=true"
+      - "traefik.http.routers.lmt-production-app.rule=Host(`${LMT_HOST_NAME}`)"
+      - "traefik.http.routers.lmt-production-app.entrypoints=websecure"
+      - "traefik.http.routers.lmt-production-app.tls=true"
+      - "traefik.http.routers.lmt-production-app.tls.certresolver=lets-encrypt-resolver"
+      - "traefik.http.routers.lmt-production-app.middlewares=response-gzip-compress@file"
```

```suggestion
      - "traefik.http.routers.lmt-production-app.middlewares=security-headers@file"
```

Gzip compression can be removed. We have to get rid of this gzip compression from traefik confgs. As I rememer, Traefik has builtin compression.

But security-headers have to be added, especially for production.